src/earn/EarnActivePools.tsx

```diff
@@ -41,51 +41,34 @@ export default function EarnActivePools() {
     [localCurrencySymbol, totalSuppliedValue]
   )
 
+  function handleTap() {
+    AppAnalytics.track(EarnEvents.earn_active_pools_cta_press, {
```

Comment:
Does it make sense to continue using this even since the action will always be "myPools"?

Reply:
Fixed in a1595f5e248e6cddf224e335bf8d7e5c2457e18b!